Plugins/Flow.Launcher.Plugin.Sys/Flow.Launcher.Plugin.Sys.csproj

```diff
@@ -37,6 +37,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Flow.Launcher.Core\Flow.Launcher.Core.csproj" />
```

Comment:
I see you brought in the core project for the default plugins, this creates a dependency on core. We need to aim to have these default plugins as self-contained plugins, where all they need will just be the plugin project, so that they are consistent with the external . net plugins. We are not there yet obviously, but I wonder is there a way to move the mssagebox code to the plugin project instead. Doing so we will also enable other .net plugins to utilise it.

Reply:
>Yeah, I have considered to export this MessageBoxEx into IPublicAPIor put it into Flow.Launcher.Plugin for all 3rd plugins to use. And in that way, Plugin.Sys do not need to refer to Core project.

Yeah exactly, and more consistency across plugins.

>By the way, Do you think it's better for me to put these two files (MessageBoxEx.xaml & MessageBoxEx.xaml.cs) in Flow.Launcher.Plugin or supply the call as a function in IPublicAPI (like MessageBoxResult ShowMessageBox(string text, string title))?

I think supply the call as a function in IPublicAPI is ideal as they are also used by flow itself, if doing this makes the code complicated/tangled/messy then let's have another chat.